doc/man1/openssl-kdf.pod.in

```diff
@@ -93,8 +92,32 @@ The password must be specified for PBKDF2 and scrypt.
 
 =item B<hexpass:>I<string>
 
-Specifies the password in hexadecimal form (two hex digits per byte).
-The password must be specified for PBKDF2 and scrypt.
+As before, but
+specifies the password in hexadecimal form (two hex digits per byte).
+
+=item B<salt:>I<string>
+
+Specifies a non-secret random cryptographic salt as an alphanumeric string
+(use if it contains printable characters only).
+The length must conform to any restrictions of the KDF algorithm.
+A salt parameter is required for several KDF algorithms, such as PBKDF2.
```

Comment:
Maybe provide the link here L<EVP_KDF-PBKDF2(7)>

Reply:
done